examples/plot_0_sin.py

```diff
@@ -0,0 +1,81 @@
+# -*- coding: utf-8 -*-
+"""
+Introductory example - Plotting sin
+===================================
+
+This is a general example demonstrating a Matplotlib plot output, embedded
+rST, the use of math notation and cross-linking to other examples. It would be
+useful to compare the :download:`source Python file <plot_0_sin.py>` with the
+output below.
+
+Example files should start with a header docstring. Anything before the
+docstring is ignored by Sphinx-Gallery and will not appear in the rendered
+output, nor will it be executed. This docstring requires an rST header, used
+at the title of the example and to correctly build reference links.
+
+Once you close the docstring you will be writing Python code. This code gets
+executed by Sphinx-Gallery and any output, including plots will be captured.
+You can also alternate between code blocks and text blocks that provide
+explanations about the code.
```

Comment:
I think I will use the term text and code blocks because we've used these terms in other parts of the documentation. And I think comment is a bit confusing because you can still have comments in code blocks. But I have amended the text - thanks.

Reply:
I also specified that default behaviour is that anything following the docstring is taken as Python code, unless you add a splitter.